include/tvm/relay/doc.h

```diff
@@ -0,0 +1,435 @@
+/*!
+ *  Copyright (c) 2018 by Contributors
```

Comment:
move this to src/relay/ir/doc.h. Since this is a feature used privately by the project